Spigot-API-Patches/0055-PlayerAttemptPickupItemEvent.patch

```diff
@@ -37,6 +42,34 @@ index 00000000..2f5fdb13
 +        return item;
 +    }
 +
++    /**
++     * Gets the amount that will remain on the ground, if any
++     *
++     * @return amount that will remain on the ground
++     */
++    public int getRemaining() {
++        return remaining;
++    }
++
++    /**
++     * Set if the item will fly at the player
++     * <p>Cancelling the event will set this value to false.</p>
++     *
++     * @param flyAtPlayer True for item to fly at player
++     */
++    public void setFlyAtPlayer(boolean flyAtPlayer) {
++        this.flyAtPlayer = flyAtPlayer;
++    }
++
++    /**
++     * Gets if the item will fly at the player
++     *
++     * @return True if the item will fly at the player
++     */
++    public boolean getFlyAtPlayer() {
```

Comment:
Should have said something last month with PlayerPickuoItemEvent addition. Renaming here now would make it inconsistent.